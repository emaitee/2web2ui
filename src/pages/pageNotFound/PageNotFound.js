import React from 'react';
import { EmptyState } from '@sparkpost/matchbox';

export default function PageNotFound() {
  return <EmptyState
    title='Page Not Found'
    image='Generic'>
    <p>We cannot find what you're looking for, use the navigation to find your way.</p>
  </EmptyState>;
}

