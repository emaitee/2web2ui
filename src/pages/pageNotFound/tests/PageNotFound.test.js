import { shallow } from 'enzyme';
import React from 'react';

import PageNotFound from '../PageNotFound';

describe('Page: Page Not Found tests', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<PageNotFound/>);
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
