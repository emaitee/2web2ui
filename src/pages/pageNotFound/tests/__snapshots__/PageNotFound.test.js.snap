// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: Page Not Found tests renders correctly 1`] = `
<EmptyState
  image="Generic"
  title="Page Not Found"
>
  <p>
    We cannot find what you're looking for, use the navigation to find your way.
  </p>
</EmptyState>
`;
